common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/OpenIDProviderConfiguration.java

```diff
@@ -0,0 +1,26 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+import java.net.URL;
+import java.util.ArrayList;
+
+/*
+ * Represents the information returned from the OpenID Provider Configuration Endpoint
+ * https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderMetadata
+ */
+
+public class OpenIDProviderConfiguration {
+
+    private String mIssuer;
+    private URL mAuthorizationEndpoint;
+    private URL mTokenEndpoint;
+    private URL mUserInfoEndpoint;
+    private URL mJWKSUri;
+    private ArrayList<String> mSupportedScopes;
```

Comment:
Can we change adjust the types here to be the interface `List<String>` instead of the concrete type `ArrayList<String>` ?

Reply:
Yep.  Good point.